### PR TITLE
Add more shl tests

### PR DESCRIPTION
**Pull request by pirapira**
Adding more shl tests from https://github.com/ethereum/EIPs/blob/master/EIPS/eip-145.md

### CODE REVIEW

File: src/GeneralStateTestsFiller/stShift/shlffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff-ffFiller.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                        "storage" : {
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                "balance" : "1000000000000000000",
+                "code" : "PUSH32 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff; PUSH1 0xff; SHL; PUSH1 0; SSTORE",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stShift/shlffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff-ffFiller.json]: comment fields should be prefixed with //

Comment (pirapira) [reply src/GeneralStateTestsFiller/stShift/shlffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff-ffFiller.json]: I'll fix it in my open PRs.

File: src/GeneralStateTestsFiller/stShift/shlffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff-ffFiller.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
+{
+    "shlffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff-ff" : {
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stShift/shlffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff-ffFiller.json]: the test name is too long. better place the desciption into
```
"_info" : 
{
    "comment" : "description of a test"
}
```

Comment (pirapira) [reply src/GeneralStateTestsFiller/stShift/shlffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff-ffFiller.json]: Any suggestion for naming conventions?

**Comment by winsvega** [comment]
this PR should not be green as there are double fields in .json without // 
and test names are too long and contain `-`  in its name

the same goes to other PRs as well.

**Comment by pirapira** [comment]
@winsvega I created an issue about double fields in `.json` without `//` https://github.com/ethereum/tests/issues/425

**Comment by pirapira** [comment]
@winsvega why shall I avoid `-`?  Shall I use `_` instead?

**Comment by pirapira** [comment]
@winsvega how many characters are allowed in a name of a test?

**Comment by winsvega** [comment]
we haven't decided about the characters. As I implemented test name length restriction I found out that there are plenty of tests with long names > 60 chars. 
We wanted it to be at least < 50.  or even shorter. 

Long test names are inconvinient when you use --singletest option in console to select such a test for execution. And it might be that if you clone the repo git will complain about long file paths.

**Comment by pirapira** [comment]
The discussion about the max length: https://github.com/ethereum/tests/issues/426

File: src/GeneralStateTestsFiller/stShift/shlffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff-ffFiller.json (hunk 2 above)
Comment (pirapira) [reply src/GeneralStateTestsFiller/stShift/shlffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff-ffFiller.json]: Anyway, the comments don't appear in the output of `ls`.  It's easier to find a descriptive file name.

**Comment by pirapira** [comment]
I'll rename the tests into `shl_-1_1Filler.json` style.

**Comment by pirapira** [comment]
Renamed tests.

**Comment by winsvega** [comment]
I need to run testeth on cpp:develop to see that tests are working.

**Comment by pirapira** [comment]
@winsvega shall I open a PR on cpp:develop or do you?

**Comment by winsvega** [comment]
what with cpp::testeth ? it should be same as cpp::develop for testeth? 
could we test it with cpp::testeth repo (cause it was the purpose of that repo)

**Comment by pirapira** [comment]
Yes, `ethereum/testeth` can test this branch.

**Comment by pirapira** [comment]
CircleCI is running at https://github.com/ethereum/testeth/pull/19

**Comment by pirapira** [comment]
@winsvega Travis runs fine.

**Pull request merged**